atomkraft/test/__init__.py

```diff
@@ -39,7 +42,9 @@ def trace(
     Test blockchain by running one trace
     """
 
-    test_trace(trace.name, reactor if reactor is None else reactor.name, keypath)
+    test_trace(
+        trace.name, reactor if reactor is None else reactor.name, keypath, verbose
```

Comment:
`reactor if reactor is None else reactor.name` 
Is this correct? This is equivalent to `None if reactor is None else reactor.name`

Reply:
I did not write this line. I had to modify it for the call to add the `verbose` argument. But you are right. I will take a look.

Reply:
It is fine. It avoids accessing `reactor.name` if `reactor` is `None`.